File: Python Excel Examples/WorkbookApiDemo/createWorkbookFromSource.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.workbook_api.WorkbookApi(configuration)

name = "CreateWorkbookFromSource.xlsx"
source_path="ExcelDocument"
source_password=""
source_storage=""
password = ""
storage = ""
folder = "Workbook"
api.create_workbook_from_source(name, source_path=source_path, source_password=source_password, source_storage=source_storage, password=password, storage=storage, folder=folder)

File: PHP Excel Examples/WorkbookApiDemo/createWorkbookFromSource.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorkbookApi($configuration);

$name = "FromSource.xlsx";
$sourcePath = "input";
$sourcePassword = "Password";
$sourceStorage = null;
$password = "123";
$storage = null;
$folder = "output";
$result = $apiInstance->CreateWorkbookFromSource($name, $sourcePath, $sourceStorage, $sourceStorage, $password, $storage, $folder);